stylesheets/archetype/ui/_breakpoints.scss

```diff
@@ -2,18 +2,28 @@
 
 // wraps the content block in a breakpoint media query
 // @mixin     breakpoint
-// @param     $key {String} the breakpoint to use
+// @param     $key {String|List} the breakpoint(s) to use
 // @content
 @mixin breakpoint($key) {
```

Comment:
thanks for taking this on!

consider doing something like...

```scss
// wraps the content block in a breakpoint media query
// @mixin     breakpoint
// @param     $key... {String} the breakpoint(s) to use
// @content
@mixin breakpoint($key...) {
  $breakpoints: ();
  
  @each $name in $key {
    $breakpoint: get-breakpoint($name);
    
    // if we got a valid breakpoint, keep it...
    @if $breakpoint {
      $breakpoints: append($breakpoints, $breakpoint, comma);
    }
    // othwerwise, throw a warning...
    @else {
      @warn "[archetype:breakpoint] could not find a breakpoint for `#{$name}`";
    }
  }
  
  // output all of our collected breakpoints
  @if length($breakpoints) > 0 {
    @media #{$breakpoints} {
      /* [archetype:breakpoint:begin] --- #{$key} --- */
      @content;
      /* [archetype:breakpoint:end] --- #{$key} --- */
    }
  }
}
```

this includes two optimizations:
1) uses a single parameter `$key...` which can take a variable number of arguments, this simplifies the API to `@include breakpoint(key1, key2, key3)` as opposed to having to pass a list literal
2) we keep track of the resolved breakpoints in a comma delimited list, which Sass' `@media` directive is able to handle natively